### PR TITLE
CU_#860q6d11w added script to download data files

### PR DESCRIPTION
**Pull request by Ayomikuolatunji**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -62,10 +63,16 @@
...
     "recharts": "^2.4.1",
     "slick-carousel": "^1.8.1",
+    "ts-node": "^10.9.1",
     "uninstall": "^0.0.0"
   },
   "devDependencies": {
-    "@babel/core": "^7.20.2",
+    "@babel/cli": "^7.21.0",
+    "@babel/core": "^7.21.3",
+    "@babel/plugin-proposal-numeric-separator": "^7.18.6",
```
Comment (hhimanshu) [inline package.json:72]: Why do we need these `@babel` related plugins?

File: package.json [hunk 2]
```diff
@@ -11,15 +11,16 @@
   },
   "scripts": {
     "develop": "gatsby develop",
-    "start": "gatsby develop",
+    "start": "ts-node download-data.mjs & gatsby develop",
     "build": "gatsby build",
```
Comment (hhimanshu) [inline package.json:15]: The script also needed to be added before `gatsby build` runs

File: package.json [hunk 3]
```diff
@@ -9,17 +9,19 @@
   "engines": {
     "node": "18.x"
   },
+  "type": "module",
   "scripts": {
     "develop": "gatsby develop",
-    "start": "gatsby develop",
+    "start": "ts-node --experimentalSpecifierResolution node download-data.ts & gatsby develop",
```
Comment (hhimanshu) [inline package.json:15]: Why are you not calling `download-data` here?

File: wget-log.1 [hunk 4]
```diff
@@ -0,0 +1,5 @@
+138.68.34.161
```
Comment (hhimanshu) [inline wget-log.1:1]: Why this file is added?

File: wget-log [hunk 5]
```diff
@@ -0,0 +1,5 @@
+138.68.34.161
```
Comment (hhimanshu) [inline wget-log:1]: Why this file is added?

File: download-data.ts [hunk 6]
```diff
@@ -0,0 +1,44 @@
+import { $ } from "zx";
```
Comment (hhimanshu) [inline download-data.ts:1]: Move this file to a new top-level directory called `scripts`

File: wget-log (hunk 5 above)
Comment (Ayomikuolatunji) [reply wget-log:1]: It was automatically added whenever you run the script. I have added it to gitignore

File: wget-log.1 (hunk 4 above)
Comment (Ayomikuolatunji) [reply wget-log.1]: removed

File: download-data.ts (hunk 6 above)
Comment (Ayomikuolatunji) [reply download-data.ts]: done

File: package.json (hunk 3 above)
Comment (Ayomikuolatunji) [reply package.json]: done

**Review by hhimanshu — APPROVED** [review]
LGTM!

**Pull request merged**